sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,14 @@
+-- select "actorId"
+-- from "actors"
+-- where "firstName" = 'Lisa';
+
+select
+-- "filmId",
+--         "fg"."genreId" as "genre id",
+        "g"."name" as "genre",
+        count(*) as "total count"
+from "castMembers"
+join "filmGenre" as "fg" using ("filmId")
+join "genres" as "g" using ("genreId")
+where "actorId" = 178
```

Comment:
Regardless of the current record ids in the database, your query should only count films for the given actor by-name:

---

![image](https://user-images.githubusercontent.com/16143771/180911823-63eaa051-29cf-473c-b3ce-1ab9dcbb428b.png)